exercises/trajectories/taskspace_iris.ipynb

```diff
@@ -268,7 +268,8 @@
     "    Returns:\n",
     "        np.ndarray: Shape (2,), 2D coordinate of the closest point on the\n",
     "            obstacle to the ellipse\n",
-    "        float: Distance from this closest point to the ellipse center\n",
+    "        float: Distance between the closest point on the ball-space obstacle \n",
```

Comment:
i don't understand "ball-space obstacle"?  do you mean the distance in the ellipsoidal metric?  or the standard euclidean distance?

Reply:
I suppose I mean in the ellipsoidal metric? The distance from $\tilde{v}_{j,k}$ to the boundary of the unit ball, where $\tilde{v}_{j,k} = C^{-1}(v_{j, k} - d)$ (I'm calling $\tilde{v}_{j, k}$ the "point on the ball-space obstacle", apologies for the imprecise terminology).

Reply:
I still don't quite understand.  What does the unit ball have to do with it?  I think there is a well-defined question as the value of the ellipsoid metric at a sample point (which is what I'm calling the ellipsoidal distance metric).  I think it would be better to describe it like that?